pg_search/src/api/operator/searchqueryinput.rs

```diff
@@ -14,16 +14,25 @@
 //
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
-
-use crate::api::operator::{
-    anyelement_query_input_opoid, anyelement_query_input_procoid, estimate_selectivity,
-    find_var_relation, make_search_config_opexpr_node, ReturnedNodePointer,
-};
+use crate::api::operator::{estimate_selectivity, find_var_relation, ReturnedNodePointer};
+use crate::gucs::per_tuple_cost;
+use crate::index::SearchIndex;
+use crate::postgres::index::open_search_index;
+use crate::postgres::types::TantivyValue;
 use crate::postgres::utils::locate_bm25_index;
 use crate::query::SearchQueryInput;
-use crate::schema::SearchConfig;
 use crate::{nodecast, UNKNOWN_SELECTIVITY};
-use pgrx::{pg_extern, pg_sys, AnyElement, FromDatum, Internal, PgList};
+use pgrx::{
+    check_for_interrupts, pg_extern, pg_func_extra, pg_sys, AnyElement, FromDatum, Internal,
+    PgList, PgOid, PgRelation,
+};
+use rustc_hash::FxHashSet;
+use std::ptr::NonNull;
+
+use super::{
+    anyelement_query_input_opoid, anyelement_query_input_procoid,
+    make_search_query_input_opexpr_node,
+};
 
 /// This is the function behind the `@@@(anyelement, paradedb.searchqueryinput)` operator. Since we
 /// transform those to use `@@@(anyelement, jsonb`), this function won't be called in normal
```

Comment:
similarly, not transforming to jsonb anymore

Reply:
I don't think this comment is even relevant anymore, because this is our new sequential scan function.

Reply:
hmm.  I guess that second sentence isn't, yeah.